cookbooks/aws-parallelcluster-common/resources/efa/partial/_setup.rb

```diff
@@ -16,29 +16,31 @@
 efa_tarball = "#{node['cluster']['sources_dir']}/aws-efa-installer.tar.gz"
 
 action :setup do
-  if efa_installed? && !::File.exist?(efa_tarball)
-    Chef::Log.warn("Existing EFA version differs from the one shipped with ParallelCluster. Skipping ParallelCluster EFA installation and configuration.")
-    return
-  end
+  if action_check_efa_support
+    if efa_installed? && !::File.exist?(efa_tarball)
+      Chef::Log.warn("Existing EFA version differs from the one shipped with ParallelCluster. Skipping ParallelCluster EFA installation and configuration.")
+      return
+    end
 
-  # remove conflicting packages
-  # default openmpi installation conflicts with new install
-  # new one is installed in /opt/amazon/efa/bin/
-  package conflicting_packages do
-    action :remove
-    not_if { efa_installed? }
-  end
+    # remove conflicting packages
+    # default openmpi installation conflicts with new install
+    # new one is installed in /opt/amazon/efa/bin/
+    package conflicting_packages do
+      action :remove
+      not_if { efa_installed? }
+    end
```

Comment:
I don't understand this change: looks like you added a whitespace at the beginning of the line. Could you please check?

Reply:
This is the indentation due to the new if

Reply:
Oh sorry, right!